src/Homework4/Slova.py

```diff
@@ -0,0 +1,19 @@
+import string
+
+"""Во входной строке записан текст.
+Словом считается последовательность непробельных символов идущих подряд,
+слова разделены одним или большим числом пробелов или символами конца строки.
+Определите, сколько различных слов содержится в этом тексте.
+"""
+
+# я не согласна, что не надо удалять знаки препинания, потому что,
+# если мы это не сделаем 'Пьер' и 'Пьер,' будут различными словами,
```

Comment:
According to task description -  word is a sequence of non-space characters. Yes, it means that Car and car, are different words.